13.go

```diff
@@ -836,36 +835,30 @@ func (hs *clientHandshakeState) processEncryptedExtensions(ee *encryptedExtensio
 	return nil
 }
 
-func (hs *clientHandshakeState) verifyPeerCertificate(certVerify *certificateVerifyMsg) error {
-	pub := hs.c.peerCertificates[0].PublicKey
-	_, sigType, hashFunc, err := pickSignatureAlgorithm(pub, []SignatureScheme{certVerify.signatureAlgorithm}, hs.hello.supportedSignatureAlgorithms, hs.c.vers)
+func verifyPeerCertificate(
+		certVerify *certificateVerifyMsg,
+		pubKey crypto.PublicKey,
+		signAlgosKnown []SignatureScheme,
+		transHash []byte,
+		clientServerString string) (error, alert) {
```

Comment:
`s/clientServerString/signatureContext/` or something like that?

Reply:
spec's calls this thing contextString